plugins/by-name/vim-suda/default.nix

```diff
@@ -0,0 +1,6 @@
+{ lib, ... }:
+lib.nixvim.vim-plugin.mkVimPlugin {
+  name = "vim-suda";
+  globalPrefix = "suda#";
+  maintainers = [ lib.maintainers.marcel ];
+}
```

Comment:
I'm happy to keep `settingsOptions` empty/undefined, but I wonder if it'd be nice to have a small `settingsExample` using some real-world example? Or maybe one of the examples from [upstream's docs](https://github.com/lambdalisue/vim-suda/blob/master/doc/suda.txt)?

Reply:
Considering that the prefix is not constant would it be viable to just define all options explicitly?

Reply:
Are you replying to https://github.com/nix-community/nixvim/pull/2580#discussion_r1862521732 ?

I think it'd be valid in this case to have the prefix set to `""` and do exactly that, just a little frustrating.

We'd also need to double check the auto-generated `settings` option's documentation, as it _might_ make reference to a prefix that we're not using...